src/main/java/cloud/orbit/actors/cluster/impl/RedisConnectionManager.java

```diff
@@ -140,22 +140,34 @@ public void subscribeToChannel(final String channelId, final MessageListener<Obj
     public void sendMessageToChannel(final String channelId, final Object msg)
     {
             final List<RedisOrbitClient> localMessagingClients = messagingClients.stream().filter((e) -> e.isConnectied()).collect(Collectors.toList());
-            final int activeClientCount = localMessagingClients.size();
-            if(activeClientCount > 0)
-            {
-                final int randomId = ThreadLocalRandom.current().nextInt(activeClientCount);
-                final RedissonClient client = localMessagingClients.get(randomId).getRedissonClient();
-
-                client.getTopic(channelId).publishAsync(msg).exceptionally((e) ->
-                {
-                    logger.error("Error sending message", e);
-                    return 0L;
-                });
-            }
-            else
-            {
-                throw new UncheckedException("No Redis messaging instances available.");
-            }
+            sendMessageToChannel(channelId, msg, localMessagingClients, 0);
+    }
+
+    private void sendMessageToChannel(final String channelId, final Object msg, final List<RedisOrbitClient> localMessagingClients, final int attempt) {
+        final int activeClientCount = localMessagingClients.size();
+        if(activeClientCount > 0)
+        {
+            final int randomId = ThreadLocalRandom.current().nextInt(activeClientCount);
+            final RedissonClient client = localMessagingClients.get(randomId).getRedissonClient();
+
+            client.getTopic(channelId).publishAsync(msg)
+                    .whenComplete((x, e) -> {
```

Comment:
I see with Lambda's it is nice to have short variable names but it makes it hard to understand what `x` and `e` represent. `x` being the number of clients that received the message so having a variable like `numClientsReceived` would make it easier to read the code.

Reply:
Done.